src/Database/Schema/PostgresSchema.php

```diff
@@ -409,17 +409,22 @@ public function columnSql(Table $table, $name)
 
         if (isset($data['null']) && $data['null'] === false) {
             $out .= ' NOT NULL';
+        } elseif (isset($data['null']) && $data['null'] === true) {
+            $out .= ' NULL';
```

Comment:
> Specify $data['null'] = true NULL returned.

Do we need the explicit `NULL`? We've not needed it thus far.

> Its probably best to always define the $data['null'] since who knows what database providers will want to do in the future.

I don't think the SQL standards will change that quickly. They haven't in the last 20 years at least. If we start to support a vendor that does implement null in a unique way we can address that then :smile: